keyboards/dz60/keymaps/billiams_layout2/build_flash.sh

```diff
@@ -3,7 +3,7 @@
 # dfu-programmer atmega32u4 reset
 
 # run this in the qmk_firmware directory
-make dz60:billiams
+make dz60:billiams_layout2
 dfu-programmer atmega32u4 erase --force && \
-dfu-programmer atmega32u4 flash dz60_billiams.hex && \
-dfu-programmer atmega32u4 reset
\ No newline at end of file
+dfu-programmer atmega32u4 flash dz60_billiams_layout2.hex && \
+dfu-programmer atmega32u4 reset
```

Comment:
What @drashna said. You can replace Lines 6-9 with:

```sh
make dz60:billiams_layout2:flash
```

which achieves the same result.

Reply:
ok done